src/t8_geometry/t8_geometry_base.hxx

```diff
@@ -163,6 +163,15 @@ struct t8_geometry
     return name;
   }
 
+  /**
+   * Compute the bounding box of the currently active tree.
+   */
+  virtual void
+  get_tree_bounding_box ([[maybe_unused]] const t8_cmesh_t cmesh, [[maybe_unused]] double bounds[6]) const
+  {
+    SC_ABORTF ("Tree bounding box function not implemented");
```

Comment:
Instead of aborting we could also return an error code here. This could make error handling more clean and leave room for usage like
```c++
if (compute_bounding_box) do_something;
 else do_something_else;
 ```